src/topology/continuous_function/algebra.lean

```diff
@@ -639,6 +652,24 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
```

Comment:
I don't think these belong in this section; these are not about the scalar instance in this section, but the one much higher up the file

Reply:
Done

Reply:
This instance is still in a weird place